assistop/myapp/static/css/app.css

```diff
@@ -13,7 +13,7 @@
         -ms-flex-direction: column;
             flex-direction: column;
   }
-  
+
```

Comment:
To prevent these deltas, try not to change whitespace manually. It could also be your editor changing these lines